pygin/pygin/src/panel.cpp

```diff
@@ -15,6 +37,7 @@ get_find_data::get_items(py::object const& find_data, size_t& item_count, OPERAT
 	new (mem.get()) get_find_data(py_items, result);
 	item_count = size;
 	mode = py::cast<OPERATION_MODES>(find_data["OpMode"sv]);
+	let_far_manage_py_items_life_time(std::ranges::subrange(result, result + size));
```

Comment:
Maybe span here too?

Reply:
Fixed in ba51d7b326ba6ea9b40b881b6eb1c00527f693ac.